attachment_preview/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright 2014 Therp BV (<http://therp.nl>)
```

Comment:
Please remove copyright lines from init files and change license lines (in all files) to https: